cookiecutter.json

```diff
@@ -4,6 +4,7 @@
   "github_username": "my_username",
   "project_name": "project_name",
   "description": "Short description of repository",
+  "create_requirements": "y/n",
   "version": "0.0.1",
   "language": "python",
```

Comment:
@gmmagruder-gi I wonder if we could conditionally "ask questions" i.e. if language is javascript or note, the create_requirements would be irrelevant. 
I'll see if i can maybe create a package.json file (equivalent for JS projects based on if language == node_js or something